### ISSUE TITLE
New version not working in react-native

### ISSUE THREAD

**Issue by chandu0101** [reindexio/reindex-js#5]
Hi ,
     I just upgraded reindex-js to 0.3.0 , i am getting following error while importing ..

```
TransformError: 
/node_modules/reindex-js/.babelrc.optional
See logs /var/folders/n0/c0fyqlqx0gg15mv4t5mchgj80000gn/T/react-packager.log
    at SocketClient._handleMessage (SocketClient.js:139:23)
    at BunserBuf.<anonymous> (SocketClient.js:53:42)
```

**Comment by fson** [comment]
@chandu0101 Thanks for reporting.

The problem is with latest react-native and Babel 6, which tries to read the `.babelrc` file in the installed reindex-js package and fails because it's for Babel 5. (https://github.com/facebook/react-native/issues/4062 has more info.)

As a fix I've [added `.babelrc` to `.npmignore`](https://github.com/reindexio/reindex-js/pull/6) so it doesn't get published to npm and released a new version [v0.3.1](https://github.com/reindexio/reindex-js/releases/tag/v0.3.1) to npm with this fix. Please let me know if upgrading to v0.3.1 fixes the problem for you.

**Comment by chandu0101** [comment]
its working perfect thanks :)
